src/components/ResultScreen.js

```diff
@@ -1,7 +1,26 @@
-const ResultScreen = () => (
+import PropTypes from 'prop-types';
+
+const ResultScreen = ({ totalScreen }) => (
   <div className="screen-calc">
-    0
+    { totalScreen.next || totalScreen.total || '0' }
   </div>
 );
 
+ResultScreen.propTypes = {
+  totalScreen: PropTypes.shape({
+    total: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.oneOf([null]),
+    ]),
```

Comment:
You seem to have defined a string proptype for total instead of number. Kindly use a number proptype for total since it will be receiving a number.

![image](https://github.com/lordksix/math-magic/assets/20554583/c4170edc-8de4-4cc8-9633-52dc450df373)

Reply:
@emmiiorji I have tried to recreate the error you show me, however, it is impossible

https://github.com/lordksix/math-magic/blob/e9606c7c2560479b43299962ad900d9b91f28c73/src/logic/calculate.js#L11-L15
 TOTAL, NUMBER, and OPERATION are strings.
As confirmation, every line that modifies the total is the result of the operation module which is always a string or is converted to a string :
https://github.com/lordksix/math-magic/blob/e9606c7c2560479b43299962ad900d9b91f28c73/src/logic/calculate.js#L71
https://github.com/lordksix/math-magic/blob/e9606c7c2560479b43299962ad900d9b91f28c73/src/logic/calculate.js#L85
https://github.com/lordksix/math-magic/blob/e9606c7c2560479b43299962ad900d9b91f28c73/src/logic/calculate.js#L114
https://github.com/lordksix/math-magic/blob/e9606c7c2560479b43299962ad900d9b91f28c73/src/logic/operate.js#L3-L30

I also have made the changes locally and as it is expected, it says that it should not be a number type

![Screenshot_2023-06-01_01-39-07](https://github.com/lordksix/math-magic/assets/121689749/95a8c967-d396-4285-bcc8-3a1db7aa9153)

**There are no issues and the modification suggestions breaks the code**